dotty-tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/api/keras/preprocessing/Sequence.scala

```diff
@@ -27,6 +27,6 @@ class Sequence private[api] (val underlying: PySequence) extends PythonType[PySe
       padding: Padding = Padding.Pre,
       truncating: Truncating = Truncating.Pre,
       value: Double | String = 0.0
-  ): NDArray[Long] = underlying.pad_sequences(sequences, maxLen, dtype, padding, truncating, value)
+  ): NDArray[Long] = underlying.pad_sequences(sequences, maxLen, dtype, padding, truncating, fromScalaTypesUnion(value))
```

Comment:
Whoops... so what was happening here before the fix?...

Reply:
It would freeze on this function call.

There was no way to reasonably troubleshoot this, so I thought my sad conda setup was to blame.
Then I started reading about dotty tuples and commented out the unions in this method and it worked.
After that I decided to try to unpack the tuples by hand, surprisingly defining upper bound of type `T` as `Tuple` was enough to find a `classTag` (I suppose a generic one) and that is all I need for the `match`.

I would call it a scala bug, because the upper bound shouldn't magically solve this problem :/